W-14989685 - CG 5.3 release (#84)
mulesoft/docs-accelerators: consumer-goods/5.3/modules/ROOT/pages/index.adoc

=== Trade Promotion Optimization

The https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-consumergoods/minor/{cg-version}/pages/Use%20case%204%20-%20Trade%20promotion%20optimization/[Trade Promotion Optimization^] use case allows you to visualize your promotions in one integrated platform with pre-built assets that integrate Business Intelligence Tools (BI Tools), such as CRM Analytics, into a TPM platform like Salesforce Consumer Goods Cloud.

=== POS Integration

The https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-consumer-goods/minor/{cg-version}/pages/Use%20case%206%20-%20POS%20integration/[POS Integration^] use case enables retail developers to leverage pre-built integrations to connect order and loyalty data from Point of Sale (POS) systems to Data Cloud.

=== Pricing Data integration for Trade Promotion Management

> The https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-consumer-goods/minor/{cg-version}/pages/Use%20case%207%20-%20Pricing%20Data%20Integration%20for%20TPM/[Pricing Data for TPM^] use case introduces integration of pricing data (List prices & Cost Of Goods Sold [COGS]) with Hyperforce to enable Consumer Packaged Goods companies to effectively analyze and optimize promotional activities.

Reviewer: ```suggestion
The https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-consumer-goods/minor/{cg-version}/pages/Use%20case%207%20-%20Pricing%20Data%20Integration%20for%20TPM/[Pricing Data Integration for TPM^] use case enables consumer packaged goods companies to more effectively analyze and optimize promotional activities by integrating List Price and Cost of Goods Sold data with Hyperforce.
```
